doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -589,6 +590,12 @@ Deprecated syntax and backward incompatibilities
    bi-implication in place of a simple implication. In practice, porting
    an old development to the new semantics is usually quite simple.
 
+.. cmd:: Parameter Morphism @ident : @ident
+   :name: Parameter Morphism
```

Comment:
I think this should be called `Declare Morphism` by analogy with `Declare Instance`

Reply:
and `Declare Module`

Reply:
I've no strong opinion, I can change that.

Reply:
done